docs/customresources/api.md

```diff
@@ -0,0 +1,79 @@
+# API
+
+This resource uses OpenAPI file with x-kusk annotations as the source of truth to configure routing.
+Refer to [extention](../extension.md) for the further information on how to add routing information to OpenAPI file.
+
+The required field of API resource is spec.**spec** where changed OpenAPI file is supplied as a string.
```

Comment:
```suggestion
The required field of API resource is spec.**spec** where `x-kusk`-enhanced OpenAPI file is supplied as a string. You can generate it (and integrate into your CI) using [kgw](https://github.com/kubeshop/kgw) CLI tool.
```

Reply:
`...where `x-kusk`-enhanced OpenAPI file is supplied as an embedded string. ...`